include/deal.II/matrix_free/cuda_fe_evaluation.h

```diff
@@ -135,47 +151,87 @@ namespace CUDAWrappers
      * Return the value of a finite element function at quadrature point
      * number @p q_point after a call to @p evaluate(true,...).
      */
+
```

Comment:
```suggestion
```

Reply:
You didn't delete this empty line between the documentation and the function. :wink:

Reply:
I definitely deleted a line, but I guess it was another ;) Now, its done!